include/sound/sof/ext_manifest.h

```diff
@@ -92,4 +93,10 @@ struct sof_ext_man_cc_version {
 	struct sof_ipc_cc_version cc_version;
 } __packed;
 
+struct ext_man_dbg_abi {
+	struct sof_ext_man_elem_header hdr;
+	/* use sof_ipc struct because of code re-use */
+	struct sof_ipc_user_abi_version dbg_abi;
+} __packed;
```

Comment:
ABI?

Reply:
It seems @plbossart this was a bit cryptic -- can you extend a bit on your concern?

Reply:
@ktrzcinx to confirm. This extends FW ABI with an interface to discover the debug ABI version (by parsing the ext-manifest headers). This went through the ABI review and assigned a MINOR version bump. I.e. old kernel or old fw is handled gracefully.

Reply:
@kv2019i excatly.